xml/System.Xml.Linq/XElement.xml

```diff
@@ -3657,7 +3657,7 @@ Element
 ## Remarks  
  When converting to <xref:System.Boolean> from an attribute or element, allowed values are "0", "1", and any string that produces "true" or "false" after trimming and conversion to lower case.  
   
- The equivalent method for this operator is ?qualifyHint=True&autoUpgrade=False  
+ The equivalent method for this operator is <xref:System.Xml.XmlConvert.ToBoolean(System.String)?displayProperty=fullName>.
```

Comment:
The equivalent is XmlConvert.ToBoolean(element.Value.ToLowerInvariant())), so the Value property needs to be extracted and ToLowerInvariant called; it's not a straightforward conversion. The more I think about it, the better it might be to simply delete this.